src/Mod/Fem/App/FemPostFunction.h

```diff
@@ -1,135 +1,135 @@
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#ifndef Fem_FemPostFunction_H
+#define Fem_FemPostFunction_H
+
+#include "FemPostObject.h"
+#include <App/PropertyUnits.h>
+
+#include <vtkSmartPointer.h>
+#include <vtkImplicitFunction.h>
+#include <vtkPlane.h>
+#include <vtkSphere.h>
+#include <vtkBoundingBox.h>
+
+namespace Fem
+{
+
+class AppFemExport FemPostFunction : public App::DocumentObject
+{
+    PROPERTY_HEADER(Fem::FemPostFunction);
+
+public:
+    /// Constructor
+    FemPostFunction(void);
+    virtual ~FemPostFunction();
+
+    virtual const char* getViewProviderName(void) const {
+        return "FemGui::ViewProviderFemPostFunction";
+    }
+
+    virtual App::DocumentObjectExecReturn* execute(void);
+
+    //bound box handling
+    void setBoundingBox(vtkBoundingBox b) {m_boundingBox = b;};
+
+    //get the algorithm or the data
+    vtkSmartPointer<vtkImplicitFunction> getImplicitFunction() {return m_implicit;};
+
+protected:
+    vtkSmartPointer<vtkImplicitFunction>  m_implicit;
+    vtkBoundingBox                        m_boundingBox;
+};
+
+class AppFemExport FemPostFunctionProvider : public App::DocumentObject {
+
+    PROPERTY_HEADER(Fem::FemPostFunctionProvider);
+
+public:
+    FemPostFunctionProvider(void);
+    virtual ~FemPostFunctionProvider();
+
+    virtual const char* getViewProviderName(void) const {
+        return "FemGui::ViewProviderFemPostFunctionProvider";
+    }
+
+    App::PropertyLinkList Functions;
+
+protected:
+    virtual void onChanged(const App::Property* prop);
+};
+
+
+////////////////////////////////////////////////////////////////////////////////////////////
+
+class AppFemExport FemPostPlaneFunction : public FemPostFunction
+{
+    PROPERTY_HEADER(Fem::FemPostPlaneFunction);
+
+public:
+
+    FemPostPlaneFunction(void);
+    virtual ~FemPostPlaneFunction();
+
+    App::PropertyVector           Normal;
+    App::PropertyVectorDistance   Origin;
+
+    virtual const char* getViewProviderName(void) const {
+        return "FemGui::ViewProviderFemPostPlaneFunction";
+    }
+
+protected:
+    virtual void onChanged(const App::Property* prop);
+
+    vtkSmartPointer<vtkPlane> m_plane;
+};
+
+////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I wouldn't add too many slashes here. Doxygen interprets the triple slash `///` as a documentation block, so maybe it will try to do something funny here. I don't know for sure, but I wouldn't use them.

If I wanted to create a visual block, I'd use two slashes and then hyphens or equals signs.
`// --------------------`

Reply:
0b6e79be352dcb4c49462f663bbcdc047566ce77 and 20788d16c82a7e60cc40b1fa59e838e1c53177a4